js_modules/dagit/packages/core/src/assets/LaunchAssetExecutionButton.tsx

```diff
@@ -328,7 +329,7 @@ async function stateForLaunchingAssets(
   ) {
     return {
       type: 'error',
-      error: 'Assets must be in the same repository to be materialized together.',
+      error: `Assets must be in ${repoName} to be materialized together.`,
```

Comment:
If we can refer to a specific code location in copy, do so. This allows us to avoid using a repository/workspace/location noun inaccurately. For now, this `repoAddressAsString` value will be `foo_repo@bar_location`, but when the `foo_repo` side of this is eliminated in the near future, this will just become `bar_location`.

Reply:
Got it. In terms of "eliminating" foo_repo IMO the logic should be omitting repo name when it is `__repository__` and keeping `repo_name@location_name` when it is not.

Reply:
Yeah, the `__repository__` case is something I'll be adding to the `repoAddressAsString` function shortly.